tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -439,9 +459,13 @@ export class GraphExecutor implements FunctionExecutor {
     while (stack.length > 0) {
       const promises = this.processStack(
           inputNodes, stack, context, tensorsMap, added, tensorsToKeep,
-          outputNodeNames, intermediateTensorConsumerCount, usedNodes);
+          outputNodeNames, intermediateTensorConsumerCount, usedNodes,
+          keepTensorForDebug);
       await Promise.all(promises);
     }
+    if (keepTensorForDebug) {
+      await this.printTensors(tensorsMap);
```

Comment:
Quote Yang's comment here `Can we just use env to avoid all the changes to the parameter of functions?`. 
Can we change it like below and avoid all the parameter `keepTensorForDebug`?
```
import {env} from '@tensorflow/tfjs-core';
...
if (env().getBool('MODEL_DEBUG')) {
  await this.printTensors(tensorsMap);
}

Reply:
Not sure if you and Yang are talking about the same parameters: e2e, tfjs-converter, or both?
For e2e part, I have changed parameters to MODEL_DEBUG.
For tfjs-converter, I have two considerations:
1.  getBool is not enough, we need try catch to make it work for different tfjs-core:
```
 let debug = false;
  try {
    debug = tf.env().getBool('MODEL_DEBUG');
  } catch (e) {
    console.warn(e.message);
  }
```
So I think keepTensorForDebug is simple and fast for this case.
2.  Currently there is only one line of code which access env in tfjs-converter. Even this line, I think it can be removed for current implemenation. https://github.com/tensorflow/tfjs/blob/master/tfjs-converter/src/operations/operation_mapper.ts#L438. So I think tfjs-converter can be env-free with keepTensorForDebug.

Reply:
Anyway, I used the MODEL_DEBUG in converter.